Makefile

```diff
@@ -45,16 +45,16 @@ endif
 # make version="nightly-19710702"
 # overridden version *should not* contain spaces or characters that aren't
 # legal in filesystem path names
-version=0.0.0
 ifndef version
-#  version := $(shell cat VERSION)
+  version := $(shell cat VERSION)
   ifneq ($(wildcard .git),)
     sha := $(shell git rev-parse --short HEAD)
     tag := $(version)-$(sha)
   else
     tag := $(version)
   endif
 else
+	foo := $(shell touch VERSION)
```

Comment:
Is this needed? Also, it looks like you used a tab here when this section is using spaces. You could just tabify the whole file.

Reply:
Nope, I'll update that.